packages/ckeditor5-ui/src/search/searchinfoview.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/search/searchinfoview
+*/
+
+import View from '../view';
+
+/**
+ * A view displaying an information text related to different states of {@link module:ui/search/text/searchtextview~SearchTextView}.
```

Comment:
`SearchInfoView` is a really dummy view. It displays two labels according to two observable strings and can be hidden. It does not understand any `SearchTextView` logic nor should pretend it does. I think this kind of documentation does not belong here.

Reply:
I guess the thing here is whether we approach the view semantically or functionally. In other words, whether we have "info view" -- a view to display information or "no results view" -- a view displayed when there are no results.

I think the first is more useful, especially since we display "no results" and "empty list" messages.

So, I agree with you.